aesfield/field.py

```diff
@@ -16,7 +16,7 @@ class AESField(models.TextField):
     description = 'A field that uses AES encryption.'
 
     def __init__(self, *args, **kwargs):
-        self.aes_prefix = kwargs.pop('aes_prefix', 'aes:')
+        self.aes_prefix = kwargs.pop('aes_prefix', b'aes:')
```

Comment:
I actually wonder if we should use `force_bytes` if any kind of unicode will be problematic

Reply:
https://github.com/andymckay/django-aesfield/pull/18/commits/24e6566db9b9b774791d3ae7c81de29cd0517ec8